package.json

```diff
@@ -14,12 +22,15 @@
     "node": ">=16.0.0"
   },
   "scripts": {
-    "dev": "node --experimental-json-modules ./node_modules/.bin/rollup --config rollup.config.js -w",
-    "build": "tsup --dts",
-    "docs": "typedoc --options typedoc.json --plugin typedoc-plugin-markdown  --out docs src/index.ts ",
+    "dev": "concurrently \"bun run dev:rollup\" \"bun run dev:types\"",
+    "dev:rollup": "node --experimental-json-modules ./node_modules/.bin/rollup --config rollup.config.js -w",
+    "dev:types": "tsup --watch",
+    "build": "tsup --dts && bun run build:rollup",
```

Comment:
build was running only tsup and no rollup. I didn't really understand why but i added rollup in the command as well